Description: unit_tests: fix missing test names
Author: xiphon <xiphon@protonmail.com>
Origin: https://github.com/monero-project/monero/pull/6346/commits/dbfa4f8
Bug: https://github.com/monero-project/monero/issues/6345
Last-Update: 2020-03-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/unit_tests/hmac_keccak.cpp
+++ b/tests/unit_tests/hmac_keccak.cpp
@@ -123,7 +123,7 @@
 }
 
 
-TEST(keccak_hmac, )
+TEST(keccak_hmac, nullptr)
 {
   test_keccak_hmac(nullptr);
 }
--- a/tests/unit_tests/keccak.cpp
+++ b/tests/unit_tests/keccak.cpp
@@ -54,10 +54,6 @@
   keccak_finish(&ctx, md1); \
   ASSERT_EQ(memcmp(md0, md1, 32), 0);
 
-TEST(keccak, )
-{
-}
-
 TEST(keccak, 0_and_0)
 {
   static const size_t chunks[] = {0};
